Sources/PrimerSDK/Classes/User Interface/TokenizationViewModels/ExternalPaymentMethodTokenizationViewModel.swift

```diff
@@ -406,14 +409,17 @@ class ExternalPaymentMethodTokenizationViewModel: PaymentMethodTokenizationViewM
         } catch {
             DispatchQueue.main.async {
                 UIApplication.shared.endIgnoringInteractionEvents()
-                PrimerDelegateProxy.checkoutFailed(with: error)
+                PrimerDelegateProxy.primerDidFailWithError(error, data: nil, decisionHandler: nil)
                 self.handleFailedTokenizationFlow(error: error)
                 self.completion?(nil, error)
             }
             return
         }
         
         firstly {
+            self.handlePrimerWillCreatePaymentEvent(PaymentMethodData(type: config.type))
```

Comment:
As I mentioned on another comment, this step should be automatically fulfilled in case the delegate function is not implemented.

Reply:
Addressed ✅